odata-protocol/11.5 Operations.md

```diff
@@ -158,7 +158,13 @@ resource whose type is the same as, or derived from, the type of the
 binding parameter of the function. The value for the binding parameter
 is the value of the resource identified by the URL prior to appending
 the function name, and additional parameter values are specified using
-[inline parameter syntax](#InlineParameterSyntax). If the function URL
+[inline parameter syntax](#InlineParameterSyntax).
+In parameters with complex or entity type, properties with a defined default
```

Comment:
Can (non-binding) parameters have an entity type? How would such an entity be specified in the function URL or in the action payload?

Reply:
For function parameters one can specify a path expression via parameter alias that resolves to an existing entity in the service, see [URL Conventions, Example 110](https://docs.oasis-open.org/odata/odata/v4.02/csd01/part2-url-conventions/odata-v4.02-csd01-part2-url-conventions.html#root).

We could allow providing a transient entity as a [complex literal](https://docs.oasis-open.org/odata/odata/v4.02/csd01/part2-url-conventions/odata-v4.02-csd01-part2-url-conventions.html#ComplexandCollectionLiterals)
- [ ] please open issue for generalizing the text and adding an example

Similar for actions with transient entities as objects in the action request body
- [ ] check text and open issue if necessary to adjust/relax text and add an an example

For path expressions in action request bodies you already opened #341.

Reply:
For transient entities, I have created #1921.

Since non-transient entities are specified through resource paths, they cannot suffer omitted properties.